ui-v2/app/components/consul-intention-form/index.hbs

```diff
@@ -1,123 +1,191 @@
+          </label>
+        {{/each}}
+      </div>
+      <label class="type-text{{if _item.error.Description ' has-error'}}">
+        <span>Description (Optional)</span>
+        <input type="text" name="Description" value={{_item.Description}} placeholder="Description (Optional)" onchange={{action 'change'}} />
+      </label>
+    </fieldset>
+    <div>
+  {{#if _item.isNew }}
+      <button type="submit" disabled={{if (or _item.isPristine _item.isInvalid (state-matches state disabledState)) 'disabled'}}>Save</button>
+  {{ else }}
+      <button type="submit" disabled={{if (or _item.isInvalid (state-matches state disabledState)) 'disabled'}}>Save</button>
+  {{/if}}
+  <button type="reset" onclick={{action oncancel _item}} disabled={{state-matches state disabledState}}>Cancel</button>
+  {{# if (and _item.ID (not-eq _item.ID 'anonymous')) }}
+      <ConfirmationDialog @message="Are you sure you want to delete this Intention?">
+        <BlockSlot @name="action" as |confirm|>
+          <button data-test-delete type="button" class="type-delete" {{action confirm dispatch "REMOVE"}} disabled={{state-matches state disabledState}}>Delete</button>
+        </BlockSlot>
+        <BlockSlot @name="dialog" as |execute cancel message|>
+          <DeleteConfirmation @message={{message}} @execute={{execute}} @cancel={{cancel}} />
+        </BlockSlot>
+      </ConfirmationDialog>
+  {{/if}}
     </div>
-    <div role="radiogroup" class={{if _item.error.Action ' has-error'}}>
-      {{#each (array 'allow' 'deny') as |intent|}}
-        <label>
-          <span>{{ capitalize intent }}</span>
-          <input type="radio" name="Action" value="{{intent}}" checked={{if (eq _item.Action intent) 'checked'}} onchange={{ action 'change' }}/>
-        </label>
-      {{/each}}
-    </div>
-    <label class="type-text{{if _item.error.Description ' has-error'}}">
-      <span>Description (Optional)</span>
-      <input type="text" name="Description" value="{{_item.Description}}" placeholder="Description (Optional)" onchange={{action 'change'}} />
-    </label>
-  </fieldset>
-  <div>
-{{#if _item.isNew }}
-    <button type="submit" disabled={{if (or _item.isPristine _item.isInvalid) 'disabled'}}>Save</button>
-{{ else }}
-    <button type="submit" disabled={{if _item.isInvalid 'disabled'}}>Save</button>
-{{/if}}
-    <button type="reset" onclick={{action oncancel _item}}>Cancel</button>
-{{# if (and _item.ID (not-eq _item.ID 'anonymous')) }}
-    <ConfirmationDialog @message="Are you sure you want to delete this Intention?">
-      <BlockSlot @name="action" as |confirm|>
-        <button data-test-delete type="button" class="type-delete" {{action confirm ondelete _item}}>Delete</button>
-      </BlockSlot>
-      <BlockSlot @name="dialog" as |execute cancel message|>
-        <DeleteConfirmation @message={{message}} @execute={{execute}} @cancel={{cancel}} />
-      </BlockSlot>
-    </ConfirmationDialog>
-{{/if}}
-  </div>
-</form>
-
+  </form>
```

Comment:
All of this green is just the previous form we already had placed inside the StateChart.